app/services/email_alert_api/subscriber_list_updater.rb

```diff
@@ -43,6 +54,14 @@ def bulk_unsubscribe
       Services.email_alert_api.bulk_unsubscribe(**args)
     end
 
+    def item_base_path_sym
+      @item_base_path_sym ||= item.base_path.to_sym
+    end
+
+    def exceptional_bulk_migration_exists?
+      EXCEPTIONAL_BULK_MIGRATIONS.key?(item_base_path_sym)
+    end
+
```

Comment:
We could simplify this code  so we don't have to check the config file twice and to make it a bit easier to read. If we have:

```
def document_collection_override_destination
   @document_collection_override_destination ||= EXCEPTIONAL_BULK_MIGRATIONS[item.base_path.to_sym]
end
```
Then on line 24 you could have
 ```
if document_collection_override_destination.present?
  check_for_some_destination_errors(document_collection_override_destination)
  bulk_migrate(subscriber_list_slug_for_document_collection)
end
```

Reply:
Done